zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/value/JobListenerEventType.java

```diff
@@ -33,5 +33,39 @@ public enum JobListenerEventType {
    * the listener should be triggered at the end of an execution, such as the completion of a
    * process instance, sub-process or element (task, event, gateway).
    */
-  END
+  END,
+
+  /**
+   * Represents the `create` event for a task listener. This event type is used to indicate that the
+   * listener should be triggered when a user task is created. This occurs when the task becomes
+   * available in the workflow for the first time.
```

Comment:
🔧 Let's refer to process > workflow

```suggestion
   * available in the process for the first time.
```

Reply:
Got it :write